views/faqs/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import React from 'react'
+import {WebView, StyleSheet} from 'react-native'
+import LoadingView from '../components/loading'
+import {text as faqs} from '../../docs/faqs.json'
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    paddingHorizontal: 30,
+    paddingTop: 10,
+    paddingBottom: 20,
+    backgroundColor: 'transparent',
+  },
+})
+
+export class FaqView extends React.Component {
+  state = {
+    html: faqs,
+  }
+
+  url = "https://stodevx.github.io/AAO-React-Native/faqs.json"
```

Comment:
Yes. The entire contents of the `docs/` folder on master is published at `https://stodevx.github.io/AAO-React-Native/`.